data/operators.csv

```diff
@@ -418,3 +418,4 @@ Rose Telecom,ISP,signed + filtering,safe,54681,24936
 Bryan Barbolina trading as Cloudwebservices,cloud,signed + filtering,safe,213268,65065
 Hazelnet,ISP,signed + filtering,safe,200242,75603
 Parknet,ISP,signed + filtering,safe,197301,19561
+Copper Valley Long Distance,ISP,signed + filtering,safe,20259,74136
```

Comment:
@Robert-CVTC we use ASRank as the ranking value. Based on the current results, it should be 76499.
https://asrank.caida.org/asns?asn=20259&type=search
```suggestion
Copper Valley Long Distance,ISP,signed + filtering,safe,20259,76499
```